packages/test/test-gc-sweep-tests/src/test/gcSweepTest.spec.ts

```diff
@@ -119,196 +84,153 @@ describeNoCompat("GC Random tests", (getTestObjectProvider) => {
 
     let overrideLogger: IgnoreErrorLogger;
 
-    let mainContainer: IContainer;
-    let connectedContainers: IContainer[];
-    let closedContainers: IContainer[];
-    let testNodes: Map<string, TestNode>;
+    const sleep = async (ms: number) => new Promise((resolve) => setTimeout(resolve, ms));
+    const seed = 0.0604202162258638; // Math.random();
+    const random: IRandom = makeRandom(seed);
 
-    const createContainer = async (): Promise<IContainer> => {
-        return provider.createContainer(runtimeFactory, { configProvider });
-    };
-    const loadContainer = async (): Promise<IContainer> => {
-        return provider.loadContainer(runtimeFactory, { configProvider });
+    const randomInteger = (min: number, max: number): number => {
+        return random.integer(min, max);
     };
 
-    const sleep = async (ms: number) => new Promise((resolve) => setTimeout(resolve, ms));
-    const seed = Math.random();
-    const random = makeRandom(seed);
+    const testTime = 10 * 1000; // 1 minute
 
-    beforeEach(async () => {
+    // Currently for GC Sweep testing only, run with npm run test. Should not be running in CI
+    // TODO: setup test to run in CI
+    it(`Planned Actions:\nSeed: ${seed}`, async () => {
```

Comment:
Did you mean to include something like `${plannedActions}` in this string?

Reply:
I got rid of planned actions, so I'll update the test name